core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.scss

```diff
@@ -56,10 +48,14 @@
 .dot-container-properties__code-loop-container {
     background: $white;
     border: 1px solid $input-border-color;
-    padding-bottom: 4rem;
 }
 
 :host ::ng-deep {
+    dot-api-link a {
+        width: $dot-api-link-width;
```

Comment:
The problem I showed is not the width is the height, this is how it looks like in the content types:

<img width="714" alt="image" src="https://user-images.githubusercontent.com/751424/205117361-b87c8d69-bb55-4650-853c-34a9e8df85c4.png">

Reply:
Noted

Reply:
Updated